Parsers/ASimAuditEvent/Parsers/vimAuditEventMicrosoftEvent.yaml

```diff
@@ -0,0 +1,259 @@
+Parser:
+    Title: Audit Event ASIM filtering parser for Microsoft Windows Events audit events
```

Comment:
couldn't find data. did we do data and filter testing using ASC for this parser?